shell.nix

```diff
@@ -0,0 +1,18 @@
+with import <nixpkgs> {};
```

Comment:
again, isn't this nixOS-specific stuff that shouldn't be committed?

Reply:
same as above